gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -345,3 +351,35 @@ nnsconf_get_fullpath (const gchar * subpluginname, nnsconf_type type)
   g_free (filename);
   return ret;
 }
+
+/** @brief Public function defined in the header */
+const gint
+nnsconf_get_value (nnsconf_type_boolean type)
+{
+  int i;
+  gint ret = -1;
+
+  switch (type) {
+    case NNSCONF_VAL_TF_MEM_OPTMZ:
+      for (i = 0; i < CONF_SOURCES; i++) {
```

Comment:
Be careful on the priorities of the three sources.
The priority should be

ENVVAR > CONFFILE > HARDCODED.